README.md

```diff
@@ -1,52 +1,52 @@
 # csdt-shashurynvitalii-2122
-Group - KI-48
-
-Student - Shashuryn Vitalii
-
-Subject - CSDT
-
-Game - Sea Battle
-
-HW i-face - I2C
-
-Data driven format - XML
-
-
-Play modes:
-
-∙ Man vs AI
-
-∙ Man vs Man
-
-∙ AI vs AI
-
-
-AI should support next complexity levels:
-
-∙ Random move;
-
-∙ Win strategy;
-
-
-Actions:
-
-∙ New;
-
-∙ Save;
-
-∙ Load;
-
-
-The proposal for data driven approach: configuration (saved in configuration file or DB):
-
-∙ board size;
-
-∙ distance;
-
-∙ tool size;
-
-∙ win statement;
-
-∙ lose statement;
-
-∙ AI parameters
+## General info about project
```

Comment:
which code is for HW?

Reply:
which code is for HW?

Reply:
second commit, server folder (serial_connection.py, server.ino files)

Reply:
how you can run py on HW? which type of HW are you using?

Reply:
I made a mistake placing both files in one folder, .py file is for client side

Reply:
so: what code is for HW?

Reply:
server.ino file for arduino